week3/homework/src/index.js

```diff
@@ -1,3 +1,85 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+
+const app = express();
+
+app.use(bodyParser.json());
+
+app.get('/todo/:id', (request, response) => {
+  fs.readFile('./todos.json', 'utf8', (error, todo) => {
+    if (error) {
+      if (error.code === 'ENOENT') {
+        console.log('no data found');
+        response.status(404).send('');
+      } else console.error(error);
+    } else {
+      const newList = todo.split('\n');
+      const id = request.params.id;
+      response.json({ result: newList[id] });
+    }
+  });
+});
+
+app.delete('/todo/:id', (request, response) => {
+  const id = request.params.id;
+  fs.readFile('./todos.json', 'utf8', (error, todosBinary) => {
+    if (error) {
+      if (error.code === 'ENOENT') {
+        console.log('no data found');
+        response.status(404).send('');
```

Comment:
Same comment as above. (Moving `response.status(404).send('');` to outside the `if...else`

Reply:
Done